README.md

```diff
@@ -78,6 +78,15 @@ Follow the given instructions for Login into the app.
 
 To run the tests run: `python manage.py test`.
 
+## Documentation
+
+- Join us on [Open session](https://anitab-org.zulipchat.com/#narrow/stream/237907-open-source-progs/topic/Open.20Sessions) and see [Meeting minutes](https://docs.google.com/document/d/1YF13IbBrU1ln4ZF1fOpgb-xGRgIF6tZLSjIBQgDmN7k/edit).
+- For setting up the project locally watch [this video](https://youtu.be/_b2RQGbYN9w).
+- Checkout the video of [project demonstration](https://youtu.be/3A746GppZ0Y) here.
+- Have questions in mind feel free to ask on [#question stream](https://anitab-org.zulipchat.com/#narrow/stream/237907-open-source-progs/topic/Questions).
```

Comment:
```suggestion
- [Design and Mocks Google Drive folder](https://drive.google.com/drive/folders/1MybSH3f8peXGUSRxhDydDtoAi8WJL1th)
- [Timeline deliverables for the project during GSoC 2020](https://docs.google.com/document/d/1xl9F5kMZrKo4mNhnP0SKpk7WkQc8PLca1ym7EZMpjSc/edit)
- [GSoC 2020 Project Meeting Minutes](https://docs.google.com/document/d/1YF13IbBrU1ln4ZF1fOpgb-xGRgIF6tZLSjIBQgDmN7k/edit)
```

This can also go to the contributing section, on another PR :)

Reply:
what I suggested comes from what I wrote [here](https://anitab-org.zulipchat.com/#narrow/stream/237907-open-source-progs/topic/Open.20Sessions/near/225236028) on Zulip.

Reply:
@isabelcosta  as someone is already working on contributing section so can I suggest these changes there.